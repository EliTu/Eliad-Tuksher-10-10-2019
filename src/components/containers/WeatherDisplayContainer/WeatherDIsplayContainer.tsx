import React from 'react';
import styles from './WeatherDisplayContainer.module.css';

const WeatherDIsplayContainer: React.FC = () => {
	const { weatherDisplayCOntainerStyles } = styles;
	return <section className={weatherDisplayCOntainerStyles}></section>;
};

export default WeatherDIsplayContainer;
