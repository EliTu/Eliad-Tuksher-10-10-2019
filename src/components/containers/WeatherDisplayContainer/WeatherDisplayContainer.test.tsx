import React from 'react';
import { shallow, ShallowWrapper } from 'enzyme';
import { WeatherDisplayContainer } from './WeatherDIsplayContainer';
import initialShallowRender from '../../../utilities/test-utilities/initialShallowRender';

describe('WeatherDisplayContainer', () => {
	let component: ShallowWrapper;
	beforeEach(
		() =>
			(component = shallow(
				<WeatherDisplayContainer
					weatherError={'abc'}
					fiveDaysForecastError={'defg'}
				/>
			))
	);

	it('should render without errors', () =>
		initialShallowRender(component, '.WeatherDisplayContainerStyles'));
});
