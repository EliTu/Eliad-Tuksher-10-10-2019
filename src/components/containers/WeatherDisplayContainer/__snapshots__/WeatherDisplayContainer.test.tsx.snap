// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WeatherDisplayContainer should display the Loader componenet if isCurrentWeatherLoading is true 1`] = `
<section
  className="WeatherDisplayContainerStyles"
>
  <Loader />
</section>
`;

exports[`WeatherDisplayContainer should display the Loader componenet if isFiveDaysForecastLoading is true 1`] = `
<section
  className="WeatherDisplayContainerStyles"
>
  <Loader />
</section>
`;

exports[`WeatherDisplayContainer should render the ErrorMessage component if fiveDaysForecastError is truthy 1`] = `
<section
  className="WeatherDisplayContainerStyles"
>
  <ErrorMessage
    errorDetails="efg"
  />
</section>
`;

exports[`WeatherDisplayContainer should render the ErrorMessage component if weatherError is truthy 1`] = `
<section
  className="WeatherDisplayContainerStyles"
>
  <ErrorMessage
    errorDetails="abc"
  />
</section>
`;

exports[`WeatherDisplayContainer should render without errors 1`] = `
<section
  className="WeatherDisplayContainerStyles"
>
  <Connect(SelectedWeather) />
  <Connect(FiveDaysForecast) />
</section>
`;
